### ISSUE TITLE
buttons to move stage by specified step size

### ISSUE THREAD

**Issue by carandraug** [MicronOxford/cockpit#640]
Cockpit has the concept of step on the `StageMover` and a user can move by a step (editable via the macro stage window) via undocumented keyboard shortcuts. Lothar asked if it was possible to have buttons with arrows to move by that step distance. probably such buttons would go on the macro stage window next to the step size widgets.

**Comment by iandobbie** [comment]
Yes I think we need 6 buttons an xy set in a + arrangement for N E S W then a separate vertically stacked two which do up and down in Z.
